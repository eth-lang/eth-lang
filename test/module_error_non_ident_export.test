module Main (thisisok "but not this" 230) where
