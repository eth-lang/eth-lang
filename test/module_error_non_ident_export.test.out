test/module_error_non_ident_export.test:1:20 Expected module export, got "but not this" of type 'string'
